nitrocli/nitrocli.1

```diff
@@ -0,0 +1,30 @@
+.TH NITROCLI 1
+.SH NAME
+nitrocli \- access Nitrokey devices
+.SH SYNOPSIS
+.B nitrocli
+\fBclear\fR|\fBclose\fR|\fBopen\fR|\fBstatus\fR
+.SH DESCRIPTION
```

Comment:
It appears all `man` pages that I've looked into include what appears to be the date of creation and the program version. E.g., for `sed` I can see:
`TH SED "1" "March 2018" "sed 4.5" "User Commands"`

Should we include that as well?

Reply:
We could. I left it out as I did not see a clear advantage and as it is an additional field to maintain.  If we add it, we should use the format `YYYY-MM-DD` for the date to conform to `man-pages(7)`. I’d leave out the source and manual fields as they have sensible defaults.

Reply:
I added the date.